lib/response.js

```diff
@@ -520,5 +522,12 @@ module.exports = {
       message: this.message,
       header: this.header
     };
+  },
+
+  /**
+   * Flush any set headers, and begin the body
+   */
+  flushHeaders: function(){
+    this.res.write('');
```

Comment:
@PlasmaPower, what is the exact version of Node.js you are using, so I can know for sure what is at line 190 in `_http_server.js`?